TANMEDIAGROUP/SYNDEX_BN
# Ft-roles&hierarcy requests

* * *

### What does this PR do?

This pull request adds an admin/all route to the codebase, and also introduces a new "publishers" feature. Additionally, it changes the initial hashing algorithm used and passes the authenticated user through the application.

### Description of Task to be completed?

Detailed description of the work done in this PR:

#### Tasks

- [x] Added an admin/all route to the application
- [x] Implemented the publishers feature, allowing creation, updating, and deletion of publisher entities
- [x] Changed the initial hashing algorithm from SHA-256 to bcrypt
- [x] Ensured that the authenticated user is passed through the application

### How to test

- Run the application locally
- Visit the `/admin/all` route and verify that it is accessible
- Create, update, and delete publisher entities to test the new functionality
- Inspect the code changes to ensure the hashing algorithm and user passing is implemented correctly

### Any background context you want to provide?

The admin/all route was added to provide a centralized request for administrators to view and manage all aspects of the application. The publisher's feature was introduced to allow better organization and categorization of content. The hashing algorithm change was made to improve security, and passing the authenticated user will enable better personalization and access control throughout the application.

### What are the relevant Github Issues if any?

null

#### Tracker ID:
null

* * *
